templates/social-template.svg

```diff
@@ -24,12 +24,12 @@
     <image x="5" y="3" width="{{=it.logoWidth}}" height="14" xlink:href="{{=it.logo}}"/>
   {{?}}
   <g fill="#333" text-anchor="middle" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-weight="700" font-size="110px" line-height="14px">
-    <text x="{{=((it.widths[0]+it.logoWidth+it.logoPadding)/2)*10}}" y="150" fill="#fff" transform="scale(0.1)" textLength="{{=(it.widths[0]-(10+it.logoWidth+it.logoPadding))*10}}" lengthAdjust="spacing">{{=it.escapeXml(it.capitalize(it.text[0]))}}</text>
-    <text x="{{=((it.widths[0]+it.logoWidth+it.logoPadding)/2)*10}}" y="140" transform="scale(0.1)" textLength="{{=(it.widths[0]-(10+it.logoWidth+it.logoPadding))*10}}" lengthAdjust="spacing">{{=it.escapeXml(it.capitalize(it.text[0]))}}</text>
+    <text x="{{=((it.widths[0]+it.logoWidth+it.logoPadding)/2)*10}}" y="150" fill="#fff" transform="scale(0.1)" textLength="{{=(it.widths[0]-(10+it.logoWidth+it.logoPadding))*10}}" lengthAdjust="spacing">{{=it.escapedText[0]}}</text>
+    <text x="{{=((it.widths[0]+it.logoWidth+it.logoPadding)/2)*10}}" y="140" transform="scale(0.1)" textLength="{{=(it.widths[0]-(10+it.logoWidth+it.logoPadding))*10}}" lengthAdjust="spacing">{{=it.escapedText[0]}}</text>
```

Comment:
I moved the capitalization code into `makeBadge`, fixing a bug that caused the width to be computed on the uncapitalized label instead of the capitalized label. This may affect the appearance of the left badges slightly. @RedSparr0w not sure if you think we should adjust something else to compensate.

Reply:
Nice, Didn't realise the labels where being capitalized in the template for these.
It looks like the letters on the right are closer together? Think it may just be the deployment server though

You can definitely see the difference with `ll` as the label
Current:
<img src="https://img.shields.io/badge/ll-social-green.svg?longCache=false&style=social" height="40px"/>
This PR:
<img src="https://shields-staging-pr-1405.herokuapp.com/badge/ll-social-green.svg?longCache=false&style=social" height="40px"/>

Don't think it needs any other changes, should be good now.

Reply:
Yea, the font is all wrong on the staging server.

I'm thinking we should maybe think about switching to a different font on staging. The badges look really bad when they're displayed with Verdana after being measured using Helvetica.